docker/Dockerfile

```diff
@@ -16,5 +16,9 @@ RUN set -ex; \
 
 RUN set -ex; \
     pip3 --no-cache-dir install --no-deps ldap2pg; \
+    mkdir -p /workspace; \
```

Comment:
fyi, [`WORKDIR` instruction](https://docs.docker.com/engine/reference/builder/#workdir) creates the directory.

Reply:
Thanks for the point ! :-)